pkg/kn/commands/domain/domain.go

```diff
@@ -44,6 +44,7 @@ func NewDomainCommand(p *commands.KnParams) *cobra.Command {
 
 type RefFlags struct {
 	reference string
+	tls       string
```

Comment:
happy to keep it, but maybe `certSecret` would be more appropriate ?

Reply:
I think `tls` might be better because it corresponds to the flag name (`--tls`).